travis-compose.yml

```diff
@@ -16,6 +18,6 @@ services:
     links:
       - db
   db:
-    image: postgres:10.5-alpine
+    image: postgres:11-alpine
```

Comment:
Is this going to be a problem when we deploy to production / heroku?  I'm getting this when I do `docker-compose up`:

```
db_1   | 2019-05-10 00:49:38.112 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 10, which is not compatible with this version 11.2.
qualitasserver_db_1 exited with code 1
```

Reply:
yea... you'll need to do `docker-compose up -V` this will recreate the volumes.